services/common/type/baseWaiver.js

```diff
@@ -0,0 +1,41 @@
+export const baseWaiver = {
+  packageGroup: "waiver",
+  componentType: "waivernew",
+  typeLabel: "1915(b) Base Waiver",
+  idType: "waiver",
+  idLabel: "Waiver Number",
+  idRegex: "^[A-Z]{2}[.-][0-9]{4,5}.R00.00$",
+  idExistValidations: [
+    {
+      idMustExist: false,
+      errorLevel: "error",
+    },
+  ],
+  allowMultiplesWithSameId: false,
+  requiredAttachments: [],
+  optionalAttachments: [
+    "1915(b)(4) FFS Selective Contracting (Streamlined) waiver application pre-print (Initial, Renewal, Amendment)",
+    "1915(b) Comprehensive (Capitated) Waiver Application Pre-print (Initial, Renewal, Amendment)",
+    "1915(b) Comprehensive (Capitated) Waiver Cost effectiveness spreadsheets (Initial, Renewal, Amendment)",
+    "1915(b)(4) FFS Selective Contracting (Streamlined) and 1915(b) Comprehensive (Capitated) Waiver Independent Assessment (first two renewals only)",
+    "Tribal Consultation (Initial, Renewal, Amendment)",
+    "Other",
+  ],
+  deprecatedAttachmentTypes: ["Required Upload (per Waiver Authority)"],
+  requiredUploads: [], // until changed in front end
+  optionalUploads: [
```

Comment:
This is more a question born from a lack of knowledge, but why are there two similar collections in optionalAttachments and optionalUploads?

Reply:
the duplication of fields is to support the front end's current needs - left as a TODO for when we add the rest of the forms...